pages/create/group/[type].tsx

```diff
@@ -48,11 +91,7 @@ const CreateGroup = () => {
           {available && (
```

Comment:
hello @ihorkitrum , since we are doing the safe check on the server side, now we can remove the

`available` from the client side, and also remove the useEffect, and construct the default state with the properties received from the SSR